src/main.cpp

```diff
@@ -377,7 +377,7 @@ int main ( int argc, char** argv )
                                "--norecord" ) )
         {
             bDisableRecording = true;
-            qInfo() << "- recording will not be enabled";
+            qInfo() << "- recording will not be enabled on startup";
```

Comment:
Might even be better to say "recording will not take place until enabled", which is what it means.  I don't like the "on startup" -- it's not really about "on startup" - it's until further action.

Reply:
Done.